docs/bastion.md

```diff
@@ -0,0 +1,72 @@
+# Bastion: Axlearn's Job Scheduler
+
+## Control Flow of Job Submission
+```mermaid
+%% elk seems to be more maintained, see: https://github.com/mermaid-js/mermaid/issues/1969
+%% N.B. elk doesn't stop rendering invisible edge operator, i.e. ~~~
+%%{init: {"flowchart": {"defaultRenderer": "elk"}} }%%
+
+flowchart TB
+
+    subgraph UserMachine ["User's dev machine (e.g. MacBook Pro)"]
+        subgraph AXLearnGithubRepository["AXLearn Github Repository"]
```

Comment:
Actually, submit need not start from github repo. We should be able to launch from a pip install in a different CWD.